src/argilla_sdk/records/_io/_common.py

```diff
@@ -12,3 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from argilla_sdk.records._io._generic import GenericIOMixin  # noqa
+from argilla_sdk.records._io._json import JSONIOMixin  # noqa
+from argilla_sdk.records._io._datasets import HFDatasetsIOMixin  # noqa
+
+
+class RecordsIOMixin(GenericIOMixin, JSONIOMixin, HFDatasetsIOMixin):
```

Comment:
I'm not a big fan of multi-inheritance. Times I've used I get a bit lost in the code. And also, I think we can refactor this a bit to do need this multi-inheritance :-)